cmd/gateway-azure.go

```diff
@@ -304,12 +342,24 @@ func (a *azureObjects) ListObjects(bucket, prefix, marker, delimiter string, max
 			ContentEncoding: object.Properties.ContentEncoding,
 		})
 	}
+
+	// Remove minio.sys.tmp prefix.
+	for i, prefix := range resp.BlobPrefixes {
+		if prefix == "minio.sys.tmp" {
+			resp.BlobPrefixes = append(resp.BlobPrefixes[:i], resp.BlobPrefixes[i+1:]...)
+			break
+		}
+	}
 	result.Prefixes = resp.BlobPrefixes
 	return result, nil
 }
 
 // ListObjectsV2 - list all blobs in Azure bucket filtered by prefix
 func (a *azureObjects) ListObjectsV2(bucket, prefix, continuationToken string, fetchOwner bool, delimiter string, maxKeys int) (result ListObjectsV2Info, err error) {
+	if strings.HasPrefix(prefix, "minio.sys.tmp/") {
```

Comment:
Isn't there a static global constant for this? `gcsMinioSysTmp` ? perhaps we can have a common `gatewayMinioSysTmp` ?

Reply:
Done